config/locales/en.yml

```
@@ -25,14 +25,14 @@ en:
       within_reach_body: Multiple apps for iOS, Android, and other platforms thanks to a developer-friendly API ecosystem allow you to keep up with your friends anywhere.
       within_reach_title: Always within reach
     generic_description: "%{domain} is one server in the network"
-    hosted_on: Mastodon hosted on %{domain}
+    hosted_on: This Mastodon instance hosted by %{domain}
```

I thought Mastodon is moving away from "instance" and moving specifically to "server" as the general language?
```suggestion
    hosted_on: This Mastodon server hosted by %{domain}
```

---

I completely forgot about this, I need to redo with Eugen's direction and I'll factor in your comments as well, thanks! (I'm unaware of an "official" move from instance to server, but will happily factor that in)